src/request.rs

```diff
@@ -159,11 +159,16 @@ impl SecretType for SecretBundle {
     }
 
     fn policies(&self) -> Vec<policy::Policy> {
-        let mut policies = vec![db::get_keyset_policy(&self.request.id).unwrap()];
+        let mut policies = vec![];
+        if let std::result::Result::Ok(keyset_policy) = db::get_keyset_policy(&self.request.id) {
+            policies.push(keyset_policy);
+        }
```

Comment:
1. wouldn't it be semantically more accurate for `get_keyset_policy()` to return `Result<Option<Policy>>` -- Err if some DB stuff went wrong, Ok(None) if no policy, and Ok(Some(policy)) if the keyset-policy is found. I think that here a DB-error (say connection issues) will be treated as if there are no policies. (BTW the return type of `exec_first` is `Result<Option<T>>`)
2. Can we use the shorter `Ok(keyset_policy)` instead of `std::result::Result::Ok(keyset_policy)` ?

Reply:
Yeah this is probably a good idea. It makes sense to match up with the signature of the mysql methods. The only thing to consider is that most database errors probably shouldn't be exposed to the KBC. I'll look into it tomorrow.

Reply:
Ok, I added another commit to redo some of the error handling for `get_secret_policy` and `get_keyset_policy`. It's possible there is a more clever way to handle the `Result<Option<Policy>>` stuff, but I think this is decent. There are a few more tests and tweaks I want to do for `db.rs` but I'll probably leave that for another PR.